src/editors/playgroundController.ts

```diff
@@ -164,6 +169,7 @@ export default class PlaygroundController {
     this._exportToLanguageCodeLensProvider = exportToLanguageCodeLensProvider;
     this._playgroundSelectedCodeActionProvider =
       playgroundSelectedCodeActionProvider;
+    this._participantController = participantController;
```

Comment:
Why do we have this on the playgroundController class? Maybe we can remove it?

Reply:
Planned for the feature usage, but I can remove it for now. As you wish. I just fixed tests and didn't want to revert this work 🙈

Reply:
All good ! No worries!